packages/core/src/external/carequality/ihe-gateway-v2/gateways.ts

```diff
@@ -82,3 +85,7 @@ export function requiresUrnInSoapBody(gateway: XCPDGateway): boolean {
 export function requiresOnlyOneDocRefPerRequest(gateway: XCAGateway): boolean {
   return gatewaysThatAcceptOneDocRefPerRequest.includes(gateway.homeCommunityId);
 }
+
+export function isEpicGateway(gateway: XCAGateway): boolean {
```

Comment:
see this to confirm:

`SELECT * 
FROM cq_directory_entry cde 
WHERE id LIKE '1.2.840.114350.1.13%'`